docker-compose.yml

```diff
@@ -4,38 +4,34 @@ networks:
   reverse-proxy:
     external: true
     name: reverse-proxy
-  mysql:
+  mariadb:
     external: true
-    name: mysql
+    name: mariadb
   internal:
     driver: bridge
 
 services:
-  #php56:
-  #  build:
-  #    context: .docker/php56
-  #  volumes:
-  #    - ./volumes/site/www:/var/www/html
-  #    - ./.docker/php56/config/apache.conf:/etc/apache2/sites-available/000-default.conf
-  #    - ./.docker/php56/config/php.ini:/usr/local/etc/php/conf.d/custom.ini
-  #  env_file:
-  #    - .php5.env
-  #  ports:
-  #    - 8080:80
-  #  networks:
-  #    - reverse-proxy
-  #    - mysql
   wordpress:
+    depends_on:
+      - mariadb
     build:
       context: .docker/wordpress
+      args:
+        VERSION_WORDPRESS: ${VERSION_WORDPRESS:-6.4}
+        VERSION_PHP: ${VERSION_PHP:-8.3}
     restart: unless-stopped
     volumes:
       - ./volumes/wordpress:/var/www/html
     env_file:
       - .env
+    environment:
+      WORDPRESS_DEBUG: "true"
+      PHP_ERROR_REPORTING: "E_ALL"
+      PHP_DISPLAY_ERRORS: "On"
```

Comment:
Maybe we can remove it.
cc: @vitormattos @Any97Cris

Reply:
Will you maintain?